advice_animal/api.py

```diff
@@ -31,6 +35,9 @@ def __init__(self, path: Path) -> None:
         # A relative path (or None) to a top-level python dir
         self.top_level_dir: Optional[Path] = infer_top_level_dir(path)
 
+        self.repo_root = self._find_repo_root()
+        self.py_projects = self.find_py_projects()
```

Comment:
Is it an error if `py_projects` is empty list?

Reply:
Are you asking if we should raise an error if we can't find any py_projects?